p2p/pex_reactor.go

```diff
@@ -47,18 +52,22 @@ type PEXReactor struct {
 	book              *AddrBook
 	ensurePeersPeriod time.Duration
 
+	// Seed/Crawler Mode
+	runningSeedCrawler bool
```

Comment:
let's use either Seed or Crawler, but not both... `seedMode bool` or `crawlerMode bool`

Reply:
Okay.